plugins/inventory/linode.py

```diff
@@ -26,6 +26,14 @@
             description: marks this as an instance of the 'linode' plugin
             required: true
             choices: ['linode', 'community.general.linode']
+        ip_style:
+            description: Populate hostvars with all information available from the Linode APIv4
```

Comment:
```suggestion
            description: Populate hostvars with all information available from the Linode APIv4.
            type: string
```

Reply:
@felixfontein - I was following the established style of not ending description lines with a period as was done in the rest of this plugin.

If this is a general style thing the plugin should be doing consistently, I'll open another merge request to make updates to the other spots as well as a typo that I was waiting to fix until after this passed to avoid conflicts.

Reply:
> If this is a general style thing the plugin should be doing consistently,

It is.

> I'll open another merge request to make updates to the other spots as well as a typo that I was waiting to fix until after this passed to avoid conflicts.

Fine, as long as you fix this one in this PR so the other PR can be backported to older stable branches.

Reply:
Sounds good to me.  Thank you for the clarification @felixfontein !